factorial_cpp.cpp

```diff
@@ -0,0 +1,22 @@
+#include <iostream>
+using namespace std;
+
+int main() {
+	// your code goes here
+	int t=0,fact=1,num=0;
+	cin>>t;
+	while(t--){
+	    cin>>num;
+	    if(num==0||num==1){
+	      cout<<1<<endl;
+	    }
+	    else if(num>1){
+	         for(int i=1;i<=num;i++){
+	               fact *= i;
+	           }
+	          cout<<fact<<endl;
+	          fact = 1;
```

Comment:
Why not a local variable within the scope of while loop of test case.
`
while(t--){
int fact=1
`

Nitpick: Change it to long long

Reply:
I don't think it much affect the code right? let me correct it once too for you..Is that ok?